templates/engage/stuckstack-ebook.html

```diff
@@ -0,0 +1,31 @@
+{% extends "engage/base_engage.html" %}
+
+{% block title %}How to escape StuckStack and profit from your OpenStack investment{% endblock %}
```

Comment:
for the new engage pages we are supposed to use {% extends_with_args .... implemented by David https://github.com/canonical-web-and-design/ubuntu.com/pull/5255/files